cmd/bdjuno/desmos.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	profilestypes "github.com/desmos-labs/desmos/v2/x/profiles/types"
+	junomessages "github.com/forbole/juno/v2/modules/messages"
+)
+
+// desmosMessageAddressesParser represents a parser able to get the addresses of the involved
+// account from a Desmos message
+var desmosMessageAddressesParser = junomessages.JoinMessageParsers(
+	profilesMessageAddressesParser,
+)
+
+// profilesMessageAddressesParser represents a MessageAddressesParser for the x/profiles module
+func profilesMessageAddressesParser(_ codec.Codec, cosmosMsg sdk.Msg) ([]string, error) {
+	switch msg := cosmosMsg.(type) {
+
+	case *profilestypes.MsgRequestDTagTransfer:
+		return []string{msg.Sender, msg.Receiver}, nil
```

Comment:
hi @RiccardoM , the message parser part was what you removed ? should I remove them?

Reply:
It's fine to keep it, thanks!